tests/profiling/exporter/test_http.py

```diff
@@ -238,191 +249,180 @@ def test_export_tracer_base_path_agent_less(endpoint_test_server):
     exp.export(test_pprof.TEST_EVENTS, 0, compat.time_ns())
 
 
-def _check_tags_types(tags):
-    for k, v in tags.items():
-        assert isinstance(k, str)
-        assert isinstance(v, bytes)
+def _parse_tags(tags):
+    tagsMap = {}
+    for tag in six.text_type(tags, "utf-8").split(","):
```

Comment:
I think we need to do
```suggestion
    for tag in tags.split(","):
```
?

Reply:
Yes, that what it was - and then it was failing with all those errors with smilies we were discussing. Unless I'm missing something.

Reply:
Also note that we have `parse_tags_str` in `ddtrace.internal.utils.formats` which can handle tags with URL values if needed.

Reply:
Given that this is a test - do you think we should use that here?

Reply:
I think it should be fine and saves us from reimplementing the same code.

Reply:
Should be done now